src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -193,6 +194,14 @@ const CreateImageWizard = () => {
               >
                 <RepositoriesStep />
               </WizardStep>,
+              <WizardStep
+                name="Additional packages"
```

Comment:
should it be 'Additional Red Hat packages'?

Reply:
Not in this case. This step is a bit different from all the other steps as we're merging Red Hat packages and custom packages into one step called "Additional packages" and also migrating from the `DualListSelector` to a table.